masonry/src/event.rs

```diff
@@ -30,14 +30,33 @@ pub enum WindowEvent {
     RebuildAccessTree,
 }
 
+/// An indicator of which pointer button was pressed.
+#[derive(PartialEq, Eq, Clone, Copy, Debug, Hash)]
+#[repr(u8)]
+pub enum PointerButton {
+    /// No mouse button.
+    // MUST BE FIRST (== 0)
```

Comment:
This needs a "why" here - *I* know it's because of the future `PointerButtons` type, but since that doesn't presently exist, this is actually entirely unneeded

Reply:
I've removed this comment for now and can re-add it when `PointerButtons` happens.